test/jasmine/tests/axes_test.js

```diff
@@ -3260,170 +3244,224 @@ describe('Test axes', function() {
         afterEach(destroyGraphDiv);
 
         it('should grow and shrink margins', function(done) {
+            var initialSize;
+            var previousSize;
+
+            function assertSize(msg, actual, exp) {
+                for(var k in exp) {
+                    var parts = exp[k].split(' -');
+                    var op = parts[0];
+
+                    var method = {
+                        '=': 'toBe',
+                        '~=': 'toBeWithin',
+                        grew: 'toBeGreaterThan',
+                        shrunk: 'toBeLessThan',
+                        initial: 'toBe'
+                    }[op];
+
+                    var val = op === 'initial' ? initialSize[k] : previousSize[k];
+                    var msgk = msg + ' ' + k + (parts[1] ? ' -' + parts[1] : '');
```

Comment:
What if `parts` variable does not have less than 2 indices?

Reply:
It will be ignored.